investpy/__init__.py

```diff
@@ -253,11 +263,14 @@ def get_recent_data(equity, country, as_json=False, order='ascending', debug=Fal
 
     logger.info('Searching introduced equity on Investing.com')
```

Comment:
All the changes done not related with `cross_currencies` should not be included since the previous code was properly formatted and the published changes are just modifications on how the data is tabulated or presented to the programmer. So **just the additions related to cross currencies data retrieval will be taken into consideration**, unless changes in any other function improves it.